Source/DafnyLanguageServer.Test/Various/CompilationStatusNotificationTest.cs

```diff
@@ -181,6 +181,9 @@ method Abs(x: int) returns (y: int)
       Assert.AreEqual(documentItem.Uri, completed.Uri);
       Assert.AreEqual(documentItem.Version, completed.Version);
       Assert.AreEqual(CompilationStatus.VerificationFailed, completed.Status);
+      var document = await Documents.GetVerifiedDocumentAsync(documentItem.Uri);
```

Comment:
I suggest moving this to `DiagnosticsTests`. The `CompilationStatusNotificationTest` test suite is dedicated to status notifications.

Reply:
That would be a new file, right?

Reply:
Oh I just merged the PR because I missed this comment. Feel free to do the refactoring if you think it's appropriate @cpitclaudel  and I'll approve it in another PR.

Reply:
I'll do it as soon as @camrein confirms that I didn't miss a file that would be the right place to put the test and that I should create a new one instead :)

Reply:
There's already a [DiagnosticsTest.cs](https://github.com/dafny-lang/dafny/blob/master/Source/DafnyLanguageServer.Test/Synchronization/DiagnosticsTest.cs), sorry that I didn't link it. However, I suggest creating a new test-case. It makes it easier to follow each test's intent and also makes it clear if something's broken without having to dive deep into the test.